app/lib/common/widgets/html_editor/html_editor.dart

```diff
@@ -375,7 +373,8 @@ class HtmlEditorState extends State<HtmlEditor> {
     map[ParagraphBlockKeys.type] = ParagraphBlockComponentBuilder(
       showPlaceholder: (editorState, node) => editorState.document.isEmpty,
       configuration: BlockComponentConfiguration(
-        placeholderText: (node) => widget.hintText ?? ' ',
+        placeholderText: (node) => widget.hintText ?? '',
+        padding: (node) => EdgeInsets.zero,
```

Comment:
Could also be probable reason why we see `<br>` over entering to room. And padding adjustments to keep placeholder aligned horizontally in a correct way.